src/transactionsOutcomeParsers/tokenManagementTransactionsOutcomeParser.spec.ts

```diff
@@ -0,0 +1,560 @@
+import { assert } from "chai";
+import { ErrParseTransactionOutcome } from "../errors";
+import { TokenManagementTransactionsOutcomeParser } from "./tokenManagementTransactionsOutcomeParser";
+import { SmartContractResult, TransactionEvent, TransactionLogs, TransactionOutcome } from "./resources";
+
+describe("test token management transactions outcome parser", () => {
+    const parser = new TokenManagementTransactionsOutcomeParser();
+
+    it("should test ensure error", () => {
+        const event = new TransactionEvent({
+            address: "erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u",
+            identifier: "signalError",
+            topics: ["Avk0jZ1kR+l9c76wQQoYcu4hvXPz+jxxTdqQeaCrbX8=", "dGlja2VyIG5hbWUgaXMgbm90IHZhbGlk"],
+            data: Buffer.from("QDc1NzM2NTcyMjA2NTcyNzI2Zjcy", "base64"),
+        });
+
+        const logs = new TransactionLogs({ events: [event] });
+        const scResult = new SmartContractResult({});
```

Comment:
Does the test work if we don't pass `smartContractResults`? If so, let's simplify it (better for advertising the parsers).

Reply:
It does work. Will remove empty objects.